src/Compilers/Shared/CoreClrShim.cs

```diff
@@ -53,5 +53,15 @@ internal static void RegisterProvider(object provider)
                 }
             }
         }
+
+        internal static class FileNotFoundException
```

Comment:
Yeah this file has changed since I last looked. I'm also OK with just  doing it next to use